wdl-lsp/src/server.rs

```diff
@@ -439,7 +439,11 @@ impl LanguageServer for Server {
         debug!("received `textDocument/didOpen` request: {params:#?}");
 
         if let Ok(path) = params.text_document.uri.to_file_path() {
-            if let Err(e) = self.analyzer.add_directory(path).await {
+            if let Err(e) = self
+                .analyzer
+                .add_document(path_to_uri(&path).expect("should convert to uri"))
```

Comment:
Should this just use the `text_document.uri` directly? We convert to a path then back to a URI.

Reply:
fixed!